File: 30-days-of-code/java/Day1.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Day1 {
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        Scanner scan = new Scanner(System.in);
        int inputInt = scan.nextInt();
        double inputDouble = scan.nextDouble();
        scan.nextLine();
        String inputString = scan.nextLine();
        System.out.println(i + inputInt);
        System.out.println(d + inputDouble);
        System.out.println(s.concat(inputString));
        scan.close();
    }
}

File: 30-days-of-code/go/Day1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	var i uint64 = 4
	var d float64 = 4.0
	var s string = "HackerRank "
	scanner := bufio.NewScanner(os.Stdin)
	// Declare second integer, double, and string variables
	var (
		inputInt    uint64
		inputDouble float64
		inputString string
		tmpStr      string
	)

	// Read and save an integer, double, and String to your variables.
	fmt.Scanf("%d", &inputInt)
	fmt.Scan(&tmpStr)
	inputDouble, _ = strconv.ParseFloat(tmpStr, 64)
	for scanner.Scan() {
		inputString = scanner.Text()
		break
	}
	fmt.Println(i + inputInt)
	fmt.Printf("%.1f\n", d+inputDouble)
	fmt.Printf("%s%s", s, inputString)

	if err := scanner.Err(); err != nil {
		os.Exit(1)
	}
}
